src/components/Transaction.vue

```diff
@@ -93,7 +93,9 @@ export default {
   },
   computed: {
     address() {
-      return this.inputs.length > 1 ? 'several addresses' : this.inputs[0].address
+      const addressOrigin = this.type === 'POSITIVE' ? 'inputs' : 'outputs'
+
+      return this[addressOrigin].length > 1 ? 'several addresses' : this[addressOrigin][0].address
```

Comment:
This will not work when sending transactions to someone else, because almost always there will be an additional change output. Ideally change outputs would be marked somehow, but for now a best-effort approach would be to show the first output address as "To" and assume the second one is always the change.